CHANGELOG.md

```diff
@@ -2908,30 +2928,52 @@ Additionally, thanks to [Alexander Turek][@derrabus] for consulting on the repo
+- Fixed bug [#1245] : SELF is not recognised as T_SELF token in: return new self
+- Fixed bug [#1246] : A mix of USE statements with and without braces can cause the tokenizer to mismatch brace tokens
     - Thanks to [Michał Bundyra][@michalbundyra] for the patch
-- Fixed bug #1249 : GitBlame report requires a .git directory
-- Fixed bug #1252 : Squiz.Strings.ConcatenationSpacing fix creates syntax error when joining a number to a string
-- Fixed bug #1253 : Generic.ControlStructures.InlineControlStructure fix creates syntax error fixing if-try/catch
-- Fixed bug #1255 : Inconsistent indentation check results when ELSE on new line
-- Fixed bug #1257 : Double dash in CSS class name can lead to "Named colours are forbidden" false positives
-- Fixed bug #1260 : Syntax errors not being shown when error_prepend_string is set
-    - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
-- Fixed bug #1264 : Array return type hint is sometimes detected as T_ARRAY_HINT instead of T_RETURN_TYPE
+- Fixed bug [#1249] : GitBlame report requires a .git directory
+- Fixed bug [#1252] : Squiz.Strings.ConcatenationSpacing fix creates syntax error when joining a number to a string
+- Fixed bug [#1253] : Generic.ControlStructures.InlineControlStructure fix creates syntax error fixing if-try/catch
+- Fixed bug [#1255] : Inconsistent indentation check results when ELSE on new line
+- Fixed bug [#1257] : Double dash in CSS class name can lead to "Named colours are forbidden" false positives
+- Fixed bug [#1260] : Syntax errors not being shown when error_prepend_string is set
+    - Thanks to [Juliette Reinders Folmer][@jrfnl] for the patch
+- Fixed bug [#1264] : Array return type hint is sometimes detected as T_ARRAY_HINT instead of T_RETURN_TYPE
     - Thanks to [Jaroslav Hanslík][@kukulich] for the patch
-- Fixed bug #1265 : ES6 arrow function raises unexpected operator spacing errors
-- Fixed bug #1267 : Fixer incorrectly handles filepaths with repeated dir names
+- Fixed bug [#1265] : ES6 arrow function raises unexpected operator spacing errors
+- Fixed bug [#1267] : Fixer incorrectly handles filepaths with repeated dir names
     - Thanks to [Sergey Ovchinnikov][@orx0r] for the patch
-- Fixed bug #1276 : Commenting.FunctionComment.InvalidReturnVoid conditional issue with anonymous classes
-- Fixed bug #1277 : Squiz.PHP.DisallowMultipleAssignments.Found error when var assignment is on the same line as an   open tag
-- Fixed bug #1284 : Squiz.Arrays.ArrayBracketSpacing.SpaceBeforeBracket false positive match for short list syntax
+- Fixed bug [#1276] : Commenting.FunctionComment.InvalidReturnVoid conditional issue with anonymous classes
+- Fixed bug [#1277] : Squiz.PHP.DisallowMultipleAssignments.Found error when var assignment is on the same line as an open tag
+- Fixed bug [#1284] : Squiz.Arrays.ArrayBracketSpacing.SpaceBeforeBracket false positive match for short list syntax
+
```

Comment:
Same remark as for PR #388, let's keep the link lists in numeric order.

Reply:
Thanks for checking this PR. This is now fixed and the PR is ready for another review.